integration-tests/src/test/java/tachyon/client/RemoteBlockInStreamIntegrationTest.java

```diff
@@ -98,20 +99,19 @@ public RemoteBlockInStreamIntegrationTest(String dataServer, String transferType
   @After
   public final void after() throws Exception {
     mLocalTachyonCluster.stop();
-    System.clearProperty(Constants.WORKER_DATA_SERVER);
-    System.clearProperty(Constants.WORKER_NETWORK_NETTY_FILE_TRANSFER_TYPE);
-    System.clearProperty(Constants.USER_REMOTE_BLOCK_READER);
   }
 
   @Before
   public final void before() throws Exception {
+    TachyonConf conf = WorkerContext.getConf();
+    conf.set(Constants.WORKER_DATA_SERVER, mDataServerClass);
+    conf.set(Constants.WORKER_NETWORK_NETTY_FILE_TRANSFER_TYPE, mNettyTransferType);
+    conf.set(Constants.USER_REMOTE_BLOCK_READER, mRemoteReaderClass);
```

Comment:
Will setting this for the worker context conf affect these user/client parameters? Can you verify this will work?